rbac/server/api/users.py

```diff
@@ -156,6 +156,17 @@
         request.app.config.DB_CONN, request.url, user_resource, head_block
     )
 
+@USERS_BP.get("api/users/<user_id>/summary")
```

Comment:
This is a duplicate of the call above with the difference being users in url instead of user.   I believe this is done because of the need for a companion change to the front end.  Instead of doing this can you add an 's' to the url above and change the endpoint in front end?  That way it is one PR instead of 2+ and less parties involved.